docs/reference/index.rst

```diff
@@ -2,33 +2,40 @@
 Reference
```

Comment:
What do you think about adding usage docs for it as well in the same PR? 
https://zyte-spider-templates.readthedocs.io/en/latest/customization/spiders.html#customizing-parameters